src/test/java/cucumber/steps/JQueryStepDefinitions.java

```diff
@@ -0,0 +1,34 @@
+package cucumber.steps;
+
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Given;
+import io.cucumber.java.en.Then;
+import io.cucumber.java.en.When;
+import org.testng.Assert;
+import pages.jQueryUIPages.JQueryUAHomePage;
+import pages.jQueryUIPages.SearchResultPage;
+import service.DriverSingleton;
+
+public class JQueryStepDefinitions {
+    @Given("User opens JQuery site")
+    public  void  openJQueryHomePage() {
+        new JQueryUAHomePage(DriverSingleton.getDriverInstance()).pageOpener();
+    }
+
+    @When("User searches for {string}")
+    public void userSearch(String searchTerm) {
+        new JQueryUAHomePage(DriverSingleton.getDriverInstance()).fillInSearchField(searchTerm);
+    }
+
+    @And("^Opens code source$")
```

Comment:
it can be without ^$ signs, like - `@And("Opens code source")`

Reply:
changed due to your suggestion